app/views/about/privacy.en.html.erb

```diff
@@ -202,14 +202,19 @@
           protection on the address below:</p>
 
         <div class="panel panel-border-narrow">
-          Information Commissioner's Office<br/>
-          Wycliffe House, Water Lane<br/>
-          Wilmslow<br/>
-          Cheshire, SK9 5AF<br/><br/>
-          <p>
-            Tel: 0303 123 1113<br/>
-            <a href="https://www.ico.org.uk" rel="external" target="_blank">www.ico.org.uk</a>
-          </p>
+            <address>
```

Comment:
I've changed my mind a little bit here after a conversation with my FE mentor @teneightfive - sorry! Could we change this to `<div>` instead, add on the microformat data and add commas on, to create the pause needed for screen readers?